package.json

```diff
@@ -165,7 +165,7 @@
     "react-timer-mixin": "^0.13.2",
     "react-transform-hmr": "^1.0.4",
     "rebound": "^0.0.13",
-    "regenerator": "^0.8.36",
+    "regenerator-runtime": "^0.9.5",
```

Comment:
If you have control over both the compiler and the runtime, I recommend pinning exact versions, but if you don't have control over the compiler version, I think it's better to do what you're doing here: use a `^` version to allow non-breaking upgrades. I will try very hard to stick to semantic versioning, so that any `a.b.x` version of `regenerator-runtime` works with any `a.b.y` version of the compiler.